examples/monolithic_build_native/mlkem_native_monobuild.S

```diff
@@ -0,0 +1,4452 @@
+/*
+ * Copyright (c) 2024 The mlkem-native project authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/*
+ * WARNING: This file is auto-generated from scripts/autogen
+ *          Do not modify it directly.
+ */
+
+/*
+ * Monolithic assembly file bundling all assembly files within mlkem-native
+ */
+
+#include "mlkem/native/aarch64/src/intt_clean.S"
+
+/*
+ * Undo all #define directives from mlkem/native/aarch64/src/intt_clean.S
+ */
+
+/* mlkem/native/aarch64/src/intt_clean.S */
+#if defined(barrett_reduce)
+#undef barrett_reduce
+#endif
+
+/* mlkem/native/aarch64/src/intt_clean.S */
+#if defined(c_consts)
+#undef c_consts
+#endif
+
+/* mlkem/native/aarch64/src/intt_clean.S */
+#if defined(c_ninv)
+#undef c_ninv
+#endif
+
+/* mlkem/native/aarch64/src/intt_clean.S */
+#if defined(c_ninv_tw)
+#undef c_ninv_tw
+#endif
+
+/* mlkem/native/aarch64/src/intt_clean.S */
+#if defined(dup8h)
+#undef dup8h
+#endif
+
+/* mlkem/native/aarch64/src/intt_clean.S */
+#if defined(gs_butterfly)
+#undef gs_butterfly
+#endif
+
+/* mlkem/native/aarch64/src/intt_clean.S */
+#if defined(gs_butterfly_v)
+#undef gs_butterfly_v
+#endif
+
+/* mlkem/native/aarch64/src/intt_clean.S */
+#if defined(layer012_start)
+#undef layer012_start
+#endif
+
+/* mlkem/native/aarch64/src/intt_clean.S */
+#if defined(layer3456_start)
+#undef layer3456_start
+#endif
+
+/* mlkem/native/aarch64/src/intt_clean.S */
+#if defined(load_next_roots_34)
+#undef load_next_roots_34
+#endif
+
+/* mlkem/native/aarch64/src/intt_clean.S */
+#if defined(load_next_roots_56)
+#undef load_next_roots_56
+#endif
+
+/* mlkem/native/aarch64/src/intt_clean.S */
+#if defined(load_roots_012)
+#undef load_roots_012
+#endif
+
+/* mlkem/native/aarch64/src/intt_clean.S */
+#if defined(mul_ninv)
+#undef mul_ninv
+#endif
+
+/* mlkem/native/aarch64/src/intt_clean.S */
+#if defined(mulmod)
+#undef mulmod
+#endif
+
+/* mlkem/native/aarch64/src/intt_clean.S */
+#if defined(mulmodq)
+#undef mulmodq
+#endif
+
+/* mlkem/native/aarch64/src/intt_clean.S */
+#if defined(pop_stack)
+#undef pop_stack
+#endif
+
+/* mlkem/native/aarch64/src/intt_clean.S */
+#if defined(push_stack)
+#undef push_stack
+#endif
+
+/* mlkem/native/aarch64/src/intt_clean.S */
+#if defined(restore_vregs)
+#undef restore_vregs
+#endif
```

Comment:
didn't you namespace those?

Reply:
Yes, but what you undef here is the name of the macro.